core/shared/src/main/scala/org/typelevel/log4cats/ErrorLogger.scala

```diff
@@ -28,6 +28,14 @@ trait ErrorLogger[F[_]] {
 }
 
 object ErrorLogger {
+  trait Fallback[F[_]] extends ErrorLogger[F] { _: MessageLogger[F] =>
+    final def error(t: Throwable)(message: => String): F[Unit] = error(message)
+    final def warn(t: Throwable)(message: => String): F[Unit] = warn(message)
+    final def info(t: Throwable)(message: => String): F[Unit] = info(message)
+    final def debug(t: Throwable)(message: => String): F[Unit] = debug(message)
+    final def trace(t: Throwable)(message: => String): F[Unit] = trace(message)
```

Comment:
I couldn't decide whether these should be final, and in fact wasn't consistent within this PR. Any thoughts?

Reply:
I don't think there's much reason to make these `final`. These are just implementation mix-ins, there shouldn't be libraries out in the wild that specifically ask for a `Fallback` and expect it to work a certain way...